x-pack/plugin/redact/src/main/java/org/elasticsearch/xpack/redact/RedactProcessor.java

```diff
@@ -296,7 +296,8 @@ String redactMatches(byte[] utf8Bytes, String redactStartToken, String redactEnd
             var merged = mergeOverlappingReplacements(replacementPositions);
             int longestPatternName = merged.stream().mapToInt(r -> r.patternName.getBytes(StandardCharsets.UTF_8).length).max().getAsInt();
```

Comment:
`longestPatternName` probably was a name once but now its the number of bytes in the longest name. Suggest `longestPatternSize`

Reply:
👍 on renaming, but since it's the `max()` of the `length` of the `patternName`(s), I went with `maxPatternNameLength`.